### PR TITLE
[RFC] Query documentation

### PR DESCRIPTION
**Pull request by L04DB4L4NC3R**
**Inclusive functionalities**
```
    Selection
        Fuzzy Search
        Structured Search
    Count
        Base Count
        Aggregated Count
    Average
        Base Average
        Aggregated Average
    Distinct
        Base Distinct
        Distinct Count
    Limit
    Pipe
    Source
```
- [X] Hugo theme update: [e593f71](https://github.com/pathivu/pathivu-hugo-web/commit/e593f714177bbe76ba741ff006774d4719de65cd)

### CODE REVIEW

File: content/docs/query/index.md [hunk 1]
```diff
@@ -5,107 +5,469 @@ weight: 2
...
+}
+
+```
+
+Pathivu supports two types of search queries, namely fuzzy search and structured query search.
+
+<h6 id="fuzzysearch"> Fuzzy Search </h4>
 
-In the below example, we are finding the log line which containing warn.
+The `message` keyword is used for fuzzy searching in Pathivu. A simple example is given below:
```
Comment (poonai) [inline content/docs/query/index.md]: Mention fuzzy level as well.

File: content/docs/query/index.md [hunk 2]
```diff
@@ -5,107 +5,469 @@ weight: 2
...
+}
+```
+
+<h6 id="countdistinct"> Count Distinct </h4>
+
+In order to find distinct value count, you can use `distinct_count` keyword. The following command will give you a list of all distinct levels in the logs along with their count.
 
 ```sh
-distinct_count(country) as country_count
+distinct_count(level) as lvl
```
Comment (poonai) [inline content/docs/query/index.md]: same

File: content/docs/query/index.md [hunk 3]
```diff
@@ -5,107 +5,469 @@ weight: 2
...
+Distinct elements can be found, aggregated and printed using the `distinct` keyword. The distinct command also provides a feature to count the number of distinct logs matched.
+
+<h6 id="basedistinct"> Base Distinct </h4>
+
+Following the example from [count](#count), the following command will give you a list of all distinct levels in the logs. 
 
-The below query will give you all the distict country value.
 ```sh
-distinct(country) as dictinct_country
+distinct(level) as lvl
```
Comment (poonai) [inline content/docs/query/index.md]: remove lvl

Comment (L04DB4L4NC3R) [reply content/docs/query/index.md]: Fixed [0fa3a8b](https://github.com/pathivu/pathivu-hugo-web/pull/1/commits/0fa3a8b7a13c857887f9bfbb174c789f4245c1b0)

File: content/docs/query/index.md (hunk 2 above)
Comment (L04DB4L4NC3R) [reply content/docs/query/index.md]: Fixed [0fa3a8b](https://github.com/pathivu/pathivu-hugo-web/pull/1/commits/0fa3a8b7a13c857887f9bfbb174c789f4245c1b0)

File: content/docs/query/index.md (hunk 1 above)
Comment (L04DB4L4NC3R) [reply content/docs/query/index.md]: Fixed [0fa3a8b](https://github.com/pathivu/pathivu-hugo-web/pull/1/commits/0fa3a8b7a13c857887f9bfbb174c789f4245c1b0)

**Pull request merged**